models/Document/Editable/Areablock.php

```diff
@@ -51,16 +51,16 @@ class Areablock extends Model\Document\Editable implements BlockInterface
     /**
      * @internal
      *
-     * @var array
+     * @var array|null
      */
-    protected array $currentIndex;
+    protected ?array $currentIndex = null;
 
     /**
      * @internal
      *
      * @var bool
      */
-    protected bool $blockStarted;
+    protected bool $blockStarted = false;
```

Comment:
This should also be optional to prevent further errors:
`protected ?bool $blockStarted = false;`